colossus/src/main/scala/colossus/service/ServiceClient.scala

```diff
@@ -120,6 +120,8 @@ extends Controller[O,I](codec, ControllerConfig(config.pendingBufferSize, config
   private val connectionFailures    = worker.metrics.getOrAdd(Rate(name / "connection_failures"))
   private val disconnects  = worker.metrics.getOrAdd(Rate(name / "disconnects"))
   private val latency = worker.metrics.getOrAdd(Histogram(name / "latency", sampleRate = 0.10, percentiles = List(0.75,0.99)))
+  private val requestBufferRatio = worker.metrics.getOrAdd(Histogram(name / "request_buffer_ratio", sampleRate = 0.10, percentiles = List(0.75, 0.99)))
+  private val requestBufferLoad = worker.metrics.getOrAdd((Gauge(name / "request_buffer_load")))
```

Comment:
wouldn't requestBufferSize be a bit more accurate?  Splitting hairs a bit, but we really are gauging the size.